code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -50,24 +51,33 @@
 			to_chat(owner, "<span class='danger'>You feel something tearing its way out of your stomach.</span>")
 			owner.adjustToxLoss(10)
 
+/obj/item/organ/body_egg/alien_embryo/on_death()
+	. = ..()
+	if(!owner && next_stage_time) // No growing outside the body
+		COOLDOWN_RESET(src, next_stage_time)
```

Comment:
stop processing/kill/delete the organ instead

Reply:
This should do it.